temporian/core/operators/BUILD

```diff
@@ -14,7 +14,6 @@ py_library(
         ":arithmetic",
         ":assign",
         ":base",
```

Comment:
PR43 change things, but we likely still need to register all the opts here: //temporian/core/operators/calendar:day_of_month", etc.